src/app/core/services/active-user/active-user.service.ts

```diff
@@ -0,0 +1,16 @@
+import { Injectable } from '@angular/core';
+
+import { User } from '../../../core/models/user';
+
+@Injectable()
+export class ActiveUserService {
+  user: User|null = null;
```

Comment:
User | null
if you want you can add appropriate option for eslint

Reply:
fixed